All-in-one doc template idea

quaid opened the pull request:
- This includes several sections usable for one or more document types: how-to, article, FAQ, and so forth.
- A writer picks one of the templates and discards the rest of the markup.
- The templates are displayed as code blocks by indenting 4 characters; this makes the markup visible on the rendered page. To use the template, the 4 characters must be removed.
- All of this are just some ideas of how to create and maintain a single template.

Signed-off by: Karsten Wade <kwade@redhat.com> <quaid@iquaid.org>

quaid committed:
All-in-one doc template idea
- This includes several sections usable for one or more document types: how-to, article, FAQ, and so forth.
- A writer picks one of the templates and discards the rest of the markup.
- The templates are displayed as code blocks by indenting 4 characters; this makes the markup visible on the rendered page. To use the template, the 4 characters must be removed.
- All of this are just some ideas of how to create and maintain a single template.

Signed-off by: Karsten Wade <kwade@redhat.com> <quaid@iquaid.org>

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **quaid** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2Fcommunity-handbook).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/operate-first/community-handbook/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["quaid"]} -->

margarethaley commented on template-handbook-article.md:
```diff
@@ -0,0 +1,147 @@
+<!-- Title, author, date block standard in all files -->
+# Title (ex: How to Use `o1-clone` tool)
```
I would just suggest making "Use" lowercase instead, since we want the titles to be in sentence case. Other than that, I think it looks good!

margarethaley replied:
Also, maybe this document should go in the contributing folder?

quaid committed:
Making title sentence case

Signed-off by: Karsten Wade <kwade@redhat.com> <quaid@iquaid.org>

quaid committed:
Consolidating w/handbook contrib files
- This makes sense to have an easy directory with the files for contributing.
- We'll have to link some/all of it into the Community Handbook, as well.

Signed-off by: Karsten Wade <kwade@redhat.com> <quaid@iquaid.org>

margarethaley reviewed (approved):
Looks great! So sorry for the delay, I didn't see the updates until just now!
/lgtm

sesheta commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/operate-first/community-handbook/pull/16#pullrequestreview-724375342" title="LGTM">margarethaley</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2Fcommunity-handbook).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/operate-first/community-handbook/blob/main/OWNERS)~~ [margarethaley]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->
